### PR TITLE
Organize Elements Using Directories

### PR DESCRIPTION
**Pull request by switchupcb**
Using the examples and creating new elements are easier with better organization. This pull request provides better formatting for the **README** and allows the user to create (any amount of) sub-directories in their `elements` folder.

### CODE REVIEW

**Comment by switchupcb** [comment]
@Spellhammer Ready for review. Keep in mind that I am not familiar with php.

**Comment by Spellhammer** [comment]
@switchupcb Thanks for your contribution, I'll review as soon as I am able to.

**Comment by switchupcb** [comment]
It sets path to elements folder, then uses recursiveiterator to gather all .php files.

File: plugin.php [hunk 1]
```diff
@@ -17,9 +17,12 @@ function my_oxygen_elements_init()
         return;
     }
 
-    foreach ( glob(plugin_dir_path(__FILE__) . "elements/*.php" ) as $filename)
+    $path = plugin_dir_path(__FILE__) . "elements"
```
Comment (5baddi) [inline plugin.php:20]: You can keep using `glob` function just change the files pattern to `elements/**/*.php`

```php
foreach ( glob(plugin_dir_path(__FILE__) . "elements/**/*.php" ) as $filename)
{
    include $filename;
}
```

Comment (switchupcb) [reply plugin.php:20]: Does this also find files in the `"elements/*"` as well as `"elements/*/*/*...` directory?

**Comment by switchupcb** [comment]
@Spellhammer puwl wequest wednesday

**Comment by switchupcb** [comment]
https://bricksbuilder.io/ iterates faster.

**Pull request closed without merging**